create-release-notes/index.js

```diff
@@ -144,12 +144,19 @@ async function createReleaseNote(webflowToken, product, releaseName, releaseNote
         }
     }
     const createResponse = await post(webflowHost, "/v2/sites/62e28cf08913e81176ba2c39/collections/66c653f35fee4b88416da2b2/items", webflowAuthToken, body);
+    if(createResponse.errors.length > 0) {
```

Comment:
Can you add a null check here for the errors field in case it isn't there at all?